packages/components/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Changed
 
+- `Pin Code` supports 1-6 digits ([#2859](https://github.com/porsche-design-system/porsche-design-system/pull/2859))
```

Comment:
Maybe rephrase that x, y, z values are now supported for `length` prop as we usually do.

Reply:
Maybe we could add a section in our README for some common changelog entry formats.